src/status_im/subs.cljs

```diff
@@ -456,11 +457,13 @@
  :disconnected?
  :<- [:peers-count]
  :<- [:waku/v2-flag]
- (fn [[peers-count wakuv2-flag]]
-   ;; TODO Right now wakuv2 module in status-go
-   ;; does not report peer counts properly,
-   ;; so we always assume that we're connected
-   (if wakuv2-flag false (zero? peers-count))))
+ :<- [:waku/v2-peer-stats]
+ (fn [peers-count wakuv2-flag peer-stats]
```

Comment:
```suggestion
 (fn [[peers-count wakuv2-flag peer-stats]]
```

Reply:
@flexsurfer thanks!! Earlier on I tested it and it worked fine. This mistake slipped in some time ago in the previous PR: https://github.com/status-im/status-react/pull/12197/files#diff-21ed1701d49fc16290b71c7c74bfae9dc355e07910b0d4507e0d29a83aacc3bf. Looking at the code, it is clear why it seemed to `work` :)